src/Widgets/SizeLabel.vala

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2019 elementary, Inc. (https://elementary.io)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace AppCenter.Widgets {
+    public class SizeLabel : Gtk.Grid {
+        public bool using_flatpak { get; construct set; }
+        public uint64 size { get; construct set; }
+
+        private Gtk.Label size_label;
+        private Gtk.Revealer icon_revealer;
+
+        public SizeLabel (uint64 _size = 0, bool _using_flatpak = false) {
+            Object (
+                size: _size,
+                using_flatpak: _using_flatpak
+            );
+        }
+
+        construct {
+            tooltip_markup = "<b>%s</b>\n%s".printf (
+                _("Actual download size likely to be smaller."),
```

Comment:
Should this be title case maybe?

Reply:
Yeah could do title case and no punctuation since it's title-like